tests/landsat/test_commands.py

```diff
@@ -50,6 +54,20 @@ def test_transform_static_stac(self):
             item = transform_stac_to_stac(pystac.Item.from_file(stac_file))
             item.validate()
 
+    def test_transform_static_stac_asset_not_available(self):
+        """Raises an exception when geotiff asset isn't available"""
+
+        for stac_file in self.landsat_stac_files:
+            item = pystac.Item.from_file(stac_file)
+            for asset in item.get_assets():
+                # Replace a valid local file for an invalid url forcing an exception
+                asset.href = asset.href.replace(
+                    'tests/data-files/landsat/LC08_L2SR_081119_20200101_20200823_02_T2_SR_B2_small.TIF',
+                    "https://usgs.gov/just/an/invalid/url/LC08_L2SR_081119_20200101_20200823_02_T2_SR_B2_small.TIF"
+                )
+            with self.assertRaises(pystac.STACError):
+                transform_stac_to_stac(item)
+
```

Comment:
This is a good test, thanks for adding. I was also hoping to see one that shows that we do _not_ fail even though SR_B2 is missing, i.e. a test that would fail when run against the **master** branch, but would succeed against this branch.

Reply:
Morning Pete,
I'll create a test removing B2 and B10 before the transformation, so it would break for master and pass on this branch.

Reply:
@gadomski 
Test test_transform_static_stac_missing_asset_b2_b10 created!
I reckon now everything is right.